Shared/Models/TimezoneModel.swift

```diff
@@ -10,4 +10,25 @@ import Foundation
 
 struct TimezoneModel: Codable {
     
+    let dateTime: String
+    let timezoneName: String
+    let timezoneLocation: String?
+    let timezoneAbbreviation: String
+    let gmtOffset: Float
+    let isDst: Bool
+    let requestedLocation: String
+    let latitude: Float
+    let longitude: Float
```

Comment:
should these not be optional? just to be on the safe side

Reply:
The optional properties are not guaranteed to be in the JSON data that we get back from service calls, so I made it optional so decoding could continue without throwing error.  Let me know if there's another way to handle this and I'll create a ticket to revise

Reply:
I am fine with what you said, but making the other properties optional is what I am saying.
Since the backend is not owned by us and if they end up changing anything, the app will crash if it can't find the element. 

This implementation is fine too. No big deal.

Reply:
Sorry I misunderstood, thought you were asking why some of the properties were optional.  Making changes now :-)